sw/device/silicon_creator/mask_rom/mask_rom_epmp.h

```diff
@@ -59,6 +59,16 @@ void mask_rom_epmp_state_init(epmp_state_t *state, lifecycle_state_t lc_state);
  */
 void mask_rom_epmp_unlock_rom_ext_rx(epmp_state_t *state, epmp_region_t image);
 
+/**
+ * Unlocks the provided ROM_EXT image region with read-only permissions.
```

Comment:
nit: I think the comment should mention that this entry uses NAPOT.

Reply:
done

Reply:
line 67